cpp/ql/lib/experimental/Quantum/OpenSSL/AlgorithmValueConsumers/OpenSSLAlgorithmValueConsumerBase.qll

```diff
@@ -0,0 +1,9 @@
+import experimental.Quantum.Language
+import semmle.code.cpp.dataflow.new.DataFlow
+
+abstract class OpenSSLAlgorithmValueConsumer extends Crypto::AlgorithmValueConsumer instanceof Call {
```

Comment:
Why is this necessary as opposed to defining abstract classes for specific types of algorithm value consumers, as done in the JCA, where we define specific abstract library-specific AVC classes for specific algorithm types, e.g., HashAlgorithmValueConsumer?